src/test/index.js

```diff
@@ -0,0 +1,37 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+// Define the expect assertion
+const { expect } = chai;
+
+// chai middleware
+chai.use(chaiHttp);
+
+describe('INDEX js', () => {
+  it('should get home successfully', (done) => {
+    chai
+      .request(app)
+      .get('/')
+      .end((err, res) => {
+        expect(res.status).to.be.equal(200);
+        expect(res.body.status).to.be.equal('success');
+        expect(res.body.data).to.be.an('object')
+        expect(res.body.data.message).to.be.equal('Welcome to Barefoot Nomad');
+        done();
+        })
+  });
+  it('should return 404 if route not found', (done) => {
```

Comment:
there should be a space or new line between `it` block

Reply:
Alright, thank you... I will implement all the feedbacks

Reply:
done